sidebars.js

```diff
@@ -202,7 +202,14 @@ module.exports = {
         "auth_auth_module",
       ],
     },
-    "config_entries_index",
+    {
+      type: "category",
+      label: "Config entries",
+      link: {type: 'doc', id: 'config_entries_index'},
+      items: [
+        {type: 'doc', id: 'config_entries_diagnostics'},
+      ]
+    },
```

Comment:
Let's put it either under `building_integrations` or just in the root. We need to restructure this anyway, but config entries doesn't sound like a good place